server/executor/functions/function_registry.go

```diff
@@ -0,0 +1,34 @@
+package functions
+
+import "fmt"
+
+type FunctionRegistry interface {
```

Comment:
You don't need to repeat the word `Function`, it's already in the package: https://michaelwhatcott.com/go-code-that-stutters/